man/coap_session.txt.in

```diff
@@ -253,6 +258,8 @@ error.
 
 *coap_session_get_type*() returns the current session's type or 0 on error.
 
+*coap_session_is_encrypted*() returns true if the session is using OSCORE encryption, or false if plain CoAP packets are used.
+
```

Comment:
Correct return types not to be boolean.  Also, move this to after the get functions.